src/main/java/org/jabref/cli/Launcher.java

```diff
@@ -43,9 +44,15 @@ public class Launcher {
     private static Logger LOGGER;
     private static String[] ARGUMENTS;
 
+    private static boolean isDebugEnabled = false;
+
     public static void main(String[] args) {
         ARGUMENTS = args;
 
+        if (Arrays.stream(ARGUMENTS).anyMatch("--debug"::equalsIgnoreCase)) {
+            isDebugEnabled = true;
+        }
```

Comment:
```suggestion
        JabRefCLI = cli = new org.jabref.cli.JabRefCLI(args);
        isDebugEnabled = cli.isDebugLogging();
```

Reply:
Thanks! JabRefCLI throws an exception so I wrapped it in a try-catch statement.